File: Samples/General operations/CreateAndSaveDocument/VB.NET/CreateAndSaveDocument.vb
Imports System.Diagnostics
Imports System.IO

Imports BitMiracle.Docotic.Pdf

Namespace BitMiracle.Docotic.Pdf.Samples
    Public NotInheritable Class CreateAndSaveDocument
        Public Shared Sub Main()
            ' NOTE: 
            ' When used in trial mode, the library imposes some restrictions.
            ' Please visit http://bitmiracle.com/pdf-library/trial-restrictions.aspx
            ' for more information.

            Dim pathToFile As String = "CreateAndSaveDocument.pdf"

            Using pdf As New PdfDocument()

                ' let's draw "Hello, world" on the first page
                Dim firstPage As PdfPage = pdf.Pages(0)
                firstPage.Canvas.DrawString(10, 50, "Hello, world!")

                pdf.Save(pathToFile)
            End Using

            ' opens saved document in default PDF viewer
            Process.Start(pathToFile)
        End Sub
    End Class
End Namespace

File: Samples/General operations/CreateAndSaveDocument/C#/CreateAndSaveDocument.cs
using System;
using System.Diagnostics;

namespace BitMiracle.Docotic.Pdf.Samples
{
    public static class CreateAndSaveDocument
    {
        public static void Main()
        {
            // NOTE:
            // Without a license, the library won't allow you to create or read PDF documents.
            // To get a free time-limited license key, use the form on
            // https://bitmiracle.com/pdf-library/download

            LicenseManager.AddLicenseData("PUT-LICENSE-HERE");

            string pathToFile = "CreateAndSaveDocument.pdf";

            using (var pdf = new PdfDocument())
            {
                // let's draw "Hello, world" on the first page
                PdfPage firstPage = pdf.Pages[0];
                firstPage.Canvas.DrawString(10, 50, "Hello, world!");

                pdf.Save(pathToFile);
            }

            Console.WriteLine($"The output is located in {Environment.CurrentDirectory}");

            Process.Start(new ProcessStartInfo(pathToFile) { UseShellExecute = true });
        }
    }
}